LICENSE

```diff
@@ -27,7 +27,20 @@ The Chapel implementation is composed of two categories of code:
      vim              vim-based syntax coloring                    VIM
 
    third-party/
-     chpldoc-venv     python packages supporting chpldoc           see third-party/README
+     chpldoc-venv/    python packages supporting chpldoc
```

Comment:
The latter point is not meant to fall on you as part of this PR, btw. It was an afterthought....